src/Resources/doc/index.rst

```diff
@@ -0,0 +1,121 @@
+SymfonyMakerBundle
+==================
+
+Symfony Maker helps you creating empty commands, controllers, form classes,
+tests and more so you can forget about the required boilerplate code. This
+bundle is an alternative to `SensioGeneratorBundle`_ for modern Symfony
+applications and requires using Symfony 3.4 or newer and `Symfony Flex`_.
+
+Installation
+------------
+
+Run this command to install and enable this bundle in your application:
+
+.. code-block:: terminal
+
+    $ composer require maker
+
+Usage
+-----
+
+This bundle provides several commands under the ``maker:`` namespace. List them
+all executing this command:
+
+.. code-block:: terminal
+
+    $ php bin/console make:
+
+     make:command            Creates a new console command class
+     make:controller         Creates a new controller class
+     make:entity             Creates a new Doctrine entity class
+
+     [...]
+
+     make:validator          Creates a new validator and constraint class
+     make:voter              Creates a new security voter class
+
+The names of the commands are self-explanatory, but some of them include
+optional arguments and options. Check them out with the ``--help`` option:
+
+.. code-block:: terminal
+
+    $ php bin/console make:controller --help
+
+Creating your Own Makers
+------------------------
+
+In case your applications need to generate custom boilerplate code, you can
+create your own ``make: ...`` command reusing the tools provided by this bundle.
+Imagine that you need to create a ``make:report`` command. First, create an
+empty command:
+
+.. code-block:: terminal
+
+    $ cd your-project/
+    $ php bin/console make:command 'make:report'
```

Comment:
Why not create a `make:maker` command? :smiley:

Reply:
For the moment this is an "edge feature". If it gets popular, we would add it in the future.